memcached.c

```diff
@@ -10187,6 +10223,7 @@ int main (int argc, char **argv) {
 #endif
     clock_handler(0, 0, 0);
 
+#ifndef DISABLE_UNIX_SOCKET
```

Comment:
Think if an `#else` is added above to define an empty server_socket_unix() if disabled, this could be removed, since socketpath will always be null.